shielddash/core/static/js/app/api/study-api.js

```diff
@@ -5,18 +5,17 @@ import {
   getStudyFetching, getStudySuccess, getStudyFailure
 } from '../actions/study-actions';
 
-const mockEndpoints = {
-  GET_STUDIES: 'http://localhost:3009/studies',
-  GET_STUDY: 'http://localhost:3009/studys/'
+const devEndpoints = {
+  GET_STUDIES: 'http://localhost:8000/studies/',
+  GET_STUDY: 'http://localhost:8000/studies/'
 };
 
 const prodEndpoints = {
   GET_STUDIES: 'https://shielddash.herokuapp.com/studies/',
   GET_STUDY: 'https://shielddash.herokuapp.com/studies/'
 }
 
-// TODO: Check node environment to set this.
-const endpoints = process.env.NODE_ENV === 'dev' ? mockEndpoints : prodEndpoints;
+const endpoints = __PROD_API__ ? prodEndpoints : devEndpoints;
```

Comment:
I guess another alternative would be to use `process.env.PROD_API` directly here and set the value to `True` on production. What are the benefits of the webpack approach? I don't doubt you, just genuinely curious about trade-offs.